feature/qos/otg_tests/bursty_traffic_test/bursty_traffic_test.go

```diff
@@ -0,0 +1,1269 @@
+				ipHeader.Dst().SetValue(intf3.IPv4)
+				ipHeader.Priority().Dscp().Phb().SetValue(int32(data.dscp))
+
+				flow.Size().SetFixed(int32(data.frameSize))
+				flow.Rate().SetPercentage(float32(data.trafficRate))
+				flow.Duration().SetChoice("burst")
+				flow.Duration().Burst().SetPackets(int32(data.burstPackets)).SetGap(int32(data.burstMinGap))
+				flow.Duration().Burst().InterBurstGap().SetBytes(float64(data.burstGap))
+
+			}
+			ate.OTG().PushConfig(t, top)
+			ate.OTG().StartProtocols(t)
+
+			var counterNames []string
+			counters := make(map[string]map[string]uint64)
+			if !*deviations.QOSDroppedOctets {
+				counterNames = []string{
+
+					"ateOutPkts", "ateInPkts", "dutQosPktsBeforeTraffic", "dutQosOctetsBeforeTraffic",
+					"dutQosPktsAfterTraffic", "dutQosOctetsAfterTraffic", "dutQosDroppedPktsBeforeTraffic",
+					"dutQosDroppedOctetsBeforeTraffic", "dutQosDroppedPktsAfterTraffic",
+					"dutQosDroppedOctetsAfterTraffic",
+				}
+			} else {
+				counterNames = []string{
+
+					"ateOutPkts", "ateInPkts", "dutQosPktsBeforeTraffic", "dutQosOctetsBeforeTraffic",
+					"dutQosPktsAfterTraffic", "dutQosOctetsAfterTraffic", "dutQosDroppedPktsBeforeTraffic",
+					"dutQosDroppedPktsAfterTraffic",
+				}
+
+			}
+			for _, name := range counterNames {
+				counters[name] = make(map[string]uint64)
+
+				// Set the initial counters to 0.
+				for _, data := range trafficFlows {
+					counters[name][data.queue] = 0
+				}
+			}
+
+			// Get QoS egress packet counters before the traffic.
+			for _, data := range trafficFlows {
+				counters["dutQosPktsBeforeTraffic"][data.queue] = gnmi.Get(t, dut, gnmi.OC().Qos().Interface(dp3.Name()).Output().Queue(data.queue).TransmitPkts().State())
+				counters["dutQosOctetsBeforeTraffic"][data.queue] = gnmi.Get(t, dut, gnmi.OC().Qos().Interface(dp3.Name()).Output().Queue(data.queue).TransmitOctets().State())
+				counters["dutQosDroppedPktsBeforeTraffic"][data.queue] = gnmi.Get(t, dut, gnmi.OC().Qos().Interface(dp3.Name()).Output().Queue(data.queue).DroppedPkts().State())
+				//counters["dutQosDroppedOctetsBeforeTraffic"][data.queue] = gnmi.Get(t, dut, gnmi.OC().Qos().Interface(dp3.Name()).Output().Queue(data.queue).DroppedOctets().State())
+				if !*deviations.QOSDroppedOctets {
+					counters["dutQosDroppedOctetsBeforeTraffic"][data.queue] = gnmi.Get(t, dut, gnmi.OC().Qos().Interface(dp3.Name()).Output().Queue(data.queue).DroppedOctets().State())
+				}
+			}
+
+			t.Logf("Running traffic 1 on DUT interfaces: %s => %s ", dp1.Name(), dp3.Name())
+			t.Logf("Running traffic 2 on DUT interfaces: %s => %s ", dp2.Name(), dp3.Name())
+			t.Logf("Sending traffic flows: \n%v\n\n", trafficFlows)
+			ate.OTG().StartTraffic(t)
+			time.Sleep(30 * time.Second)
+			ate.OTG().StopTraffic(t)
+			time.Sleep(30 * time.Second)
+
+			for trafficID, data := range trafficFlows {
+				ateTxPkts := gnmi.Get(t, ate.OTG(), gnmi.OTG().Flow(trafficID).Counters().OutPkts().State())
```

Comment:
Change to a single query for both similar to: https://github.com/openconfig/featureprofiles/blob/e8068e3f70aeb6cbcbd454bd72337106e76c473c/feature/gribi/otg_tests/supervisor_failure_test/supervisor_failure_test.go#L163

Reply:
Done